Misc/NEWS.d/next/Library/2022-01-03-21-03-50.bpo-41011.uULmGi.rst

```diff
@@ -0,0 +1,3 @@
+Added two new variables to *pyvenv.cfg* which is generated by :mod:`venv`
+module: *executable* for the executable within the environment and *command*
```

Comment:
Not the one within the environment (you can always find that from the layout of an environment), it's the one used to create it (see my other comment).